src/Stratis.Bitcoin.FullNode.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29020.237
+# Visual Studio 15
```

Comment:
@codingupastorm what are these changes? Are they needed?

Reply:
woops I maybe used the wrong VS.